test/Governance.test.ts

```diff
@@ -490,26 +590,41 @@ describe('Governance', function () {
       proposalCalldatas,
       description
     )
+    expect(await this.bit.balanceOf(this.rewards.address)).to.equal(
+      REWARDS_ALLOCATION
+    )
 
     await this.rewards.connect(this.delegatee3).claimVotingReward(proposalId)
+    const expectedBalance: BN = this.votingPower
+      .mul(2)
```

Comment:
questions: what does this magic number mean?

There is a reason for these multiplication, but what is it? How does somebody reading the test know why this is happening?

Reply:
Will make it more clear.